### ISSUE TITLE
Chromium support on intel gpu

### ISSUE THREAD

**Issue by ghost** [ublue-os/bazzite#1622]
### Describe the bug

Hello, when using chrome://gpu in the distro's native chromium (rpm-ostree), some things are disabled:

Graphics Feature Status
=======================
*   Direct Rendering Display Compositor: Disabled
*   Raw Draw: Disabled
*   Skia Graphite: Disabled
*   Vulkan: Disabled
*   WebGPU: Disabled
*   WebNN: Disabled

In the integrated Steam browser, in the normal interface, which you access with alt+tab, during some game, some things are disabled:

Graphics Feature Status
Canvas out-of-process rasterization: Disabled
Direct Rendering Display Compositor: Disabled
Raw Draw: Disabled
Video Encode: Software only. Hardware acceleration disabled
Vulkan: Disabled
WebGPU: Disabled

### What did you expect to happen?

Compatibility to see all enabled

### Output of `rpm-ostree status`

```shell
State: idle
Deployments:
● ostree-image-signed:docker://ghcr.io/ublue-os/bazzite:stable
                   Digest: sha256:e282521ee0e418a11ae423d8cfd9c83893545c9dd2e4e7226fda1dc8f1389e6c
                  Version: 40.20240901.1 (2024-09-02T21:12:18Z)
          LayeredPackages: chromium elisa-player libreoffice vlc wine

  ostree-image-signed:docker://ghcr.io/ublue-os/bazzite:stable
                   Digest: sha256:688da1a245efc6fdae27743964c8f5ebe380f057064844c7066d3f851e92651c
                  Version: 40.20240901.0 (2024-09-02T05:05:54Z)
          LayeredPackages: chromium elisa-player libreoffice vlc wine
```

### Hardware

Laptop Lenovo Ideapad 3 15itl6 82md
intel i3 1115G4
Ram 8gb

### Extra information or context

I believe it is a bug, or compatibility. I wanted to leave the report, to help the project. 👍

**Comment by KyleGospo** [comment]
Please try the flatpak, we do not support layered browsers under any circumstance. It is a security risk to only update your browser with the OS.

**Comment by Leandro599** [comment]
Thanks. 
I installed Chrome, via the bazzite portal.
In Chrome it looks like this:

Graphics Feature Status
=======================
*   Direct Rendering Display Compositor: Disabled
*   Raw Draw: Disabled
*   Skia Graphite: Disabled
*   Video Encode: Software only. Hardware acceleration disabled
*   Vulkan: Disabled
*   WebGPU: Disabled
*   WebNN: Disabled

**Comment by KyleGospo** [comment]
You'll need this:
https://wiki.archlinux.org/title/Chromium#Hardware_video_acceleration

**Comment by Leandro599** [comment]
thanks,
I tried here, but it didn't find the .flags file here 😅

**Comment by Leandro599** [comment]
I couldn't find the flags in Chrome 😅
I uninstalled Chromium through the terminal, when I right-clicked on chromium, and in the uninstall option, when I opened discover it didn't find chromium. (sudo rpm-ostree uninstall chromium)

I did a little research here, and started Chrome with this command:
flatpak run com.google.Chrome --enable-features=VaapiVideoDecoder,VaapiIgnoreDriverChecks,Vulkan,DefaultANGLEVulkan,VulkanFromANGLE,VaapiVideoDecodeLinuxGL,ignore-gpu-blocklist,enable-zero-copy

It improved performance a lot, videos in 8k60fps were better too.

Of the disabled ones, it looked like this:
Graphics Feature Status
=======================
*   Direct Rendering Display Compositor: Disabled
*   Raw Draw: Disabled
*   Skia Graphite: Disabled
*   Video Encode: Software only. Hardware acceleration disabled
*   WebGPU: Disabled
*   WebNN: Disabled
